src/test/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTimingTest.java

```diff
@@ -157,7 +157,7 @@ public void testBasicPass() throws Exception {
         // Custom unstable status
         run.setResult(Result.UNSTABLE);
         status = StatusAndTiming.computeChunkStatus2(run, null, n[0], n[1], n[2]);
-        assertEquals(GenericStatus.UNSTABLE, status);
+        assertEquals(GenericStatus.SUCCESS, status);
```

Comment:
@svanoort Having to change this test is my biggest concern regarding potential knockons from this change.